user/soju/template.py

```diff
@@ -0,0 +1,41 @@
+pkgname = "soju"
+pkgver = "0.8.2"
+_commit = "5ee7ed56f7ed4ad0ccba13d944b5610fa46de352"
+pkgrel = 0
+build_style = "makefile"
+make_build_args = ["GOFLAGS=-buildmode=pie -tags=libsqlite3"]
+hostmakedepends = ["go", "scdoc", "libcap-progs"]
+makedepends = ["sqlite-devel"]
+pkgdesc = "User-friendly IRC bouncer"
```

Comment:
we don't use subjective adjectives for pkgdescs, should be `IRC bouncer`

`

Reply:
Yes you do.

https://github.com/chimera-linux/cports/tree/master/user/fd#L11
> `pkgdesc = "Simple, fast and user-friendly alternative to find"`

https://github.com/chimera-linux/cports/blob/master/user/pyonji/template.py#L6
> `pkgdesc = "Easy-to-use tool to send e-mail patches"`

https://github.com/chimera-linux/cports/blob/master/user/python-pyusb/template.py#L11
> `pkgdesc = "Easy USB access for Python"`